docs/install/install.rst

```diff
@@ -8,6 +8,8 @@ Note: The version definition for the HIP runtime is different from CUDA. On an A
 ``hipRuntimeGerVersion`` function returns the HIP runtime version; on an NVIDIA platform, this function
 returns the CUDA runtime version.
 
+.. _install_prerequisites:
```

Comment:
In Line 8, remove the semicolon and put in a new sentence.
On an NVIDIA platform,...

Reply:
Fixed and made it into an actual ".. note:"